src/cmd/flux-mini.py

```diff
@@ -208,6 +212,14 @@ def create_parser(self):
             default=1,
             help="Number of cores to allocate per task",
         )
+        parser.add_argument(
+            "-g",
+            "--gpus-per-task",
+            type=int,
+            metavar="N",
+            default=0,
```

Comment:
Cool!  Just reading through this, won't this fail if `-g` is not specified, since value will be zero and <1 raises an exception?

You might want to follow the pattern of `--nodes` /  `num_nodes` where it is `None` unless the user provided the option.

Reply:
Yeah, that sounds better. The assertion was a typo and should allow 0, but use of `None` feels more right to me. I think I've got it wrong anyway since the builds are failing in travis. I won't have more time to work on this until this afternoon if you'd rather get it done.

Reply:
Sure, I'll have a go.

Otherwise, are you feeling like this PR is ready?